src/MSBuild.Sdk.SqlProj/Sdk/Sdk.props

```diff
@@ -9,6 +9,8 @@
     <UsingMSBuildSdkSqlProj>true</UsingMSBuildSdkSqlProj>
     <MSBuildAllProjects Condition="'$(MSBuildToolsVersion)' != 'Current'">$(MSBuildAllProjects);$(MsBuildThisFileFullPath)</MSBuildAllProjects>
     <SqlServerVersion>Sql150</SqlServerVersion>
+    <TreatTSqlWarningsAsErrors>False</TreatTSqlWarningsAsErrors>
```

Comment:
For C# projects this is readily available as `TreatWarningsAsErrors` and has the same effect. Perhaps we should support both here to create some symmetry between project files, while still allowing people migrating from `.sqlproj` to use this.

Reply:
It looks to me like this was marked as resolved, but it wasn't implemented nor was there any comment on it. I'm going to mark unresolved; I think this suggestion makes sense.

Reply:
I've added the feedback to the place we actually need to fix this.